rust/src/data_catalog/unity/mod.rs

```diff
@@ -169,9 +167,7 @@ impl FromStr for UnityCatalogConfigKey {
 
     fn from_str(s: &str) -> Result<Self, Self::Err> {
         match s {
-            "access_token" | "unity_access_token" | "databricks_access_token" => {
-                Ok(UnityCatalogConfigKey::AccessToken)
-            }
+            "unity_token" | "databricks_token" => Ok(UnityCatalogConfigKey::Token),
```

Comment:
Hmmm can we leave the old ones in but emit a deprecation warning? Something like:

```rust
// TODO: Remove these variables
"access_token" | "unity_access_token" | "databricks_access_token" => {
    log::warn!("Environment variable '{}' is deprecated and will be remove in a future veresion. Please use UNITY_TOKEN or DATABRICKS_TOKEN instead.");
    Ok(UnityCatalogConfigKey::Token)
}
```

This will just save us some work answering questions about why their environment variables suddenly don't work anymore.

Reply:
I don't disagree but don't feel like doing that work today, so I'm going to drop this out of the milestone and release 0.16